data/dsql/sql_test.go

```diff
@@ -1,4 +1,4 @@
-package data_test
+package dsql_test
```

Comment:
Why this is in a different package? Is that necessary?

Reply:
Allows you to test and example the code as it would be when importing into a new package.